app/Resources/views/layout.html.twig

```diff
@@ -20,4 +20,5 @@
 
 {% block javascripts %}
     <script src="{{ asset('assets/js/main.js') }}"></script>
-{% endblock %}
\ No newline at end of file
+    <script src="{{ asset('assets/js/repository.js') }}"></script>
```

Comment:
:+1:
if there is a lot of file in the project, appending them to `main.js` can be tricky